### DISCUSSION TITLE
Email Plugin configuration

### DISCUSSION THREAD

**Discussion by c2006878** [nocodb/nocodb#6979] [General]
![1700038807403](https://github.com/nocodb/nocodb/assets/58558960/f7c77eac-8843-4d05-b7e0-86f6b77fe8e7)
A quick question. How do I configure the reset password function?

**Comment by o1lab** [comment]
<img width="1290" alt="image" src="https://github.com/nocodb/nocodb/assets/5435402/054b0612-bb24-4625-8a83-78221099d05a">

https://docs.nocodb.com/getting-started/self-hosted/environment-variables/
